stargate/pkg/client/client.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/proto"
+
+	pb "github.com/stargate/stargate-grpc-go-client/stargate/pkg/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type StargateClient struct {
+	client pb.StargateClient
+	conn   grpc.ClientConnInterface
+}
+
+func NewStargateClientWithConn(conn grpc.ClientConnInterface) (*StargateClient, error) {
+	client := pb.NewStargateClient(conn)
+
+	return &StargateClient{
+		client: client,
+		conn:   conn,
```

Comment:
Do we need to store `conn`? It's never used. Also, do we need the wrapper `StargateClient` anymore?

Reply:
Other than that I'm :+1:

Reply:
Good catch, I'll remove the `conn`. The wrapper is just there for convenience since everything is a method off of it. Alternative is we pass the proto client into each function.

Reply:
I'd still not prefer to have wrapper, but I digress.